ultravox/data/datasets.py

```diff
@@ -20,6 +21,7 @@
 from torch.utils import data
 
 from ultravox.data import text_proc
+from ultravox.training import config_base
```

Comment:
I think we want `training` to depend on `data`, not the other way around. I'd move `DataDictConfig` here and call it `GenericDatasetConfig` or some such.

Reply:
Good point. Will do.